package.json

```diff
@@ -5,12 +5,14 @@
   "main": "./server/app.js",
   "scripts": {
     "start": "npm run build && node ./build/app.js",
-    "db-migrate": "babel-node ./server/models/db.js",
+    "pretest": "NODE_ENV=test npm run db-migrate",
+    "test": "NODE_ENV=test mocha --require babel-core/register ./server/tests/index.js --exit --timeout 10000 --colors",
+    "db-migrate": "babel-node ./server/models/users.js",
     "build": "babel server/ --out-dir build --copy-files --source-maps",
     "babel-node": "babel-node ./server/app.js",
     "start-dev": "nodemon --exec npm run babel-node",
-    "test": "mocha --require babel-core/register ./server/tests --exit",
-    "coveralls": "nyc npm test && nyc report --reporter=text-lcov | coveralls"
+    "coveralls": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
+    "hello": "node ./server/models/new.js"
```

Comment:
Remove this. It was for debugging purpose.

Reply:
okay